Directory.Build.props

```diff
@@ -119,6 +119,9 @@
 
       See Documentation/coding-guidelines/project-guidelines.md for more details on our configurations.
     -->
+    <!-- Hack to support .NET Framework debugging from VS. Configuration maps 1:1 to TargetGroup for netfx. -->
```

Comment:
@eerhardt @weshaggard any clever idea how to support running netstandard projects with netfx from within VS? this hack only applies to projects that have a netfx configuration, which most don't have...